ansible_collections/arista/avd/tests/unit/plugins/plugin_utils/test_ipaddressing.py

```diff
@@ -0,0 +1,37 @@
+# Copyright (c) 2023 Arista Networks, Inc.
+# Use of this source code is governed by the Apache License 2.0
+# that can be found in the LICENSE file.
+import pytest
+
+from ansible_collections.arista.avd.plugins.plugin_utils.errors import AristaAvdError
+from ansible_collections.arista.avd.plugins.plugin_utils.utils import get_ip_from_pool
+
+# default values for testcases
+(pool, prefixlen, subnet_offset, ip_offset) = ("1.2.3.4/24", 32, 1, 0)
```

Comment:
probably should be capitalized to be very pedantic :p

```suggestion
pool, prefixlen, subnet_offset, ip_offset = ("1.2.3.4/24", 32, 1, 0)
```